validator_client/src/beacon_node_fallback.rs

```diff
@@ -76,30 +94,6 @@ pub fn start_fallback_updater_service<T: SlotClock + 'static, E: EthSpec>(
     Ok(())
 }
 
-/// Indicates if a beacon node must be synced before some action is performed on it.
-#[derive(PartialEq, Clone, Copy)]
-pub enum RequireSynced {
-    Yes,
-    No,
-}
-
-/// Indicates if a beacon node should be set to `Offline` if a request fails.
-#[derive(PartialEq, Clone, Copy)]
-pub enum OfflineOnFailure {
-    Yes,
-    No,
-}
-
-impl PartialEq<bool> for RequireSynced {
-    fn eq(&self, other: &bool) -> bool {
-        if *other {
-            *self == RequireSynced::Yes
-        } else {
-            *self == RequireSynced::No
-        }
-    }
-}
-
 #[derive(Debug)]
 pub enum Error<T> {
     /// The node was unavailable and we didn't attempt to contact it.
```

Comment:
Looks like `Unavailable` is no longer used?

Reply:
Fixed in [500dbe5](https://github.com/sigp/lighthouse/pull/4393/commits/500dbe54fbfdac81c14ab059cd96fc7bba53401c)